01_export.ipynb

```diff
@@ -0,0 +1,308 @@
+{
```

Comment:
<p>You need an #export tag here.</p><p> </p><p>Since this will be attached eventually to a new version of <code>learn.export</code>,  let's rework this as the following:</p><pre class="ql-syntax" spellcheck="false">@patch

def export_dls(self:Learner, fname='dls.pkl'):
  "Export fastai DataLoaders for inference"
  self._end_cleanup()
  dls = self.dls.new_empty()
  torch.save(dls, self.path/fname, pickle_protocol=2)
</pre><p> This is how I did it for fastinference and it will do the job enough. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/muellerzr/fastexport/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>